reactive/containerd.py

```diff
@@ -103,14 +105,40 @@ def charm_status():
         status.blocked('Container runtime not available')
 
 
-def merge_custom_registries(custom_registries):
+def update_custom_tls_config(config_directory, registries):
+    """
+    Read registries config and write tls files to disk.
+
+    :param str config_directory: containerd config directory
+    :param List registries: juju config for custom registries
+    :return: None
+    """
+    for registry in registries:
+        for opt in ['ca', 'key', 'cert']:
+            file_b64 = registry.get('%s_file' % opt)
+            if file_b64:
```

Comment:
if `registry['ca_file']` exist in the config.. ok, we're going to be writing tls config